py/htm/examples/rest/client.py

```diff
@@ -27,11 +27,11 @@
 #        port default: 8050
 #
 
-import requests
 import sys
-import json
 import math
 
+import rest_api
```

Comment:
if we use this, I'd suggest more explicit name. This looks like a generic 3rd-party REST library - `htm_rest_api`?